app/panel/components/BuildingBlocks/DonutGraph.jsx

```diff
@@ -159,37 +223,31 @@ class DonutGraph extends React.Component {
 			}
 		});
 
-		// Draw graph
-		const chart = select(this.node)
-			.append('svg')
-			.attr('class', 'donutSvg')
-			.attr('width', '100%')
-			.attr('height', '100%')
-			.attr('viewBox', `0 0 ${width} ${height}`)
-			.attr('preserveAspectRatio', 'xMinYMin')
-			.append('g')
-			.attr('transform', `translate(${width / 2}, ${height / 2})`);
-		const trackerArc = arc()
-			.innerRadius(radius - 13)
-			.outerRadius(radius);
-		const trackerPie = pie()
-			.startAngle(-Math.PI)
-			.endAngle(Math.PI)
-			.sort(null)
-			.value(d => d.value);
-		const g = chart.selectAll('.arc')
-			.data(trackerPie(graphData))
-			.enter().append('g')
-			.attr('class', 'arc');
+		// CONNECT NEW DATA
+		const arcs = this.chartCenter.selectAll('g')
+			.data(this.trackerPie(graphData), d => d.data.id);
+
+		// UPDATE
+		arcs.select('path')
+			.transition()
+			.duration(d => (d.value / totalTrackers) * animationDuration)
```

Comment:
I'm seeing the graph update right away.  I'm not seeing an animation transition.  Is this expected?

Reply:
The colors do change if I switch from simple to detailed view.